packages/test/test-service-load/package.json

```diff
@@ -27,7 +28,12 @@
     "lint": "npm run eslint",
     "lint:fix": "npm run eslint:fix",
     "start": "node ./dist/nodeStressTest.js",
-    "start:mini": "node ./dist/nodeStressTest.js --profile mini",
+    "start:mini": "node ./dist/nodeStressTest.js --driver tinylicious --profile mini",
+    "start:tinylicious:test": "tinylicious > tinylicious.log 2>&1",
+    "test": "npm run test:realsvc",
```

Comment:
i don;t think this works. also i don't see the test in the pr results if i search for the title "should return 0"

Reply:
i see. I just copied from the e2e package.json. which script actually triggers the CI?

Reply:
start at the root package.json, and look at which npm scripts ci:test calls, and ensure that this package has a npm script that will be called along that path

Reply:
oh! I forgot to update. I found it and it was `: report`, I added it and the test is in the pipeline now!
thanks for the help from Helio and Arin